iterators.tex

```diff
@@ -588,7 +589,7 @@
   template <class I>
   concept bool WeakIterator@\newtxt{() \{}\oldtxt{ =}@
     @\newtxt{return}@ WeaklyIncrementable<I>@\newtxt{()}@ &&
-      requires(I i) {
+      requires(@\newtxt{const }@I i) {
         { *i } -> auto&&; // pre: i is dereferenceable
```

Comment:
There's no nearby normative definition of "dereferencable", IIRC. If we make one, it should be up near Readable et al.